engine/core/src/main/java/org/datacleaner/job/concurrent/CompositeTaskListener.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * DataCleaner (community edition)
+ * Copyright (C) 2014 Neopost - Customer Information Management
+ *
+ * This copyrighted material is made available to anyone wishing to use, modify,
+ * copy, or redistribute it subject to the terms and conditions of the GNU
+ * Lesser General Public License, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution; if not, write to:
+ * Free Software Foundation, Inc.
+ * 51 Franklin Street, Fifth Floor
+ * Boston, MA  02110-1301  USA
+ */
+package org.datacleaner.job.concurrent;
+
+import org.datacleaner.job.tasks.Task;
+
+/**
+ * {@link TaskListener} that wraps several other {@link TaskListener}s
+ */
+public class CompositeTaskListener implements TaskListener {
+
+    private final TaskListener[] _listeners;
+
+    public CompositeTaskListener(TaskListener ... listeners) {
+        _listeners = listeners;
+    }
+    
+    @Override
+    public void onBegin(Task task) {
+        for (TaskListener listener : _listeners) {
+            listener.onBegin(task);
```

Comment:
We should probably wrap listeners in try/catch to avoid the whole chain dying (unless that is what we want to achieve). Same for onComplete() and onError()